app/js/features/license/saga.js

```diff
@@ -19,6 +21,7 @@ export function *doBuyLicense() {
     yield call(toSend.send, {gasLimit: estimatedGas + 1000});
     yield put({type: BUY_LICENSE_SUCCEEDED});
   } catch (error) {
+    console.error(error);
```

Comment:
Is it on purpose?

Reply:
Yes. I like having the error in the console. Don't know if you agree?

Reply:
Fine by me